source/develop/developer-guide/vdsm/developers.md

```diff
@@ -601,13 +601,13 @@ Add `import pdb; pdb.set_trace()` at the point where you want a breakpoint. Make
 services stopped. Now you have to run vdsm and supervdsm interactively. 
 First run supervdsm as in one console
 
-       /usr/bin/python3 /usr/share/vdsm/supervdsmd --sockfile /var/run/vdsm/svdsm.sock
+        $ /usr/bin/python3 /usr/share/vdsm/supervdsmd --sockfile /var/run/vdsm/svdsm.sock
```

Comment:
This would suggest that you should run 'supervdsm' as a regular user, while it needs to be run with root privileges. I think it'd be better to change '$' to '#'